app/src/main/java/com/example/composetemplate/presentation/common/LoginScreenButton.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.composetemplate.presentation.common
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.ElevatedButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import com.example.composetemplate.ui.theme.LoginButtonColorDisable
+import com.example.composetemplate.ui.theme.LoginButtonColorEnable
+
+
+@Composable
+fun LoginScreenButton(
```

Comment:
Just wondering, can't we create a general button component instead of a login specific one?

Reply:
RAZ:::: you can check if you want to do base button, now its login componnent because compose button componnet already ask for color, shape, etc....